src/lib/storage/lz4_segment/lz4_encoder.hpp

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
-#include <lib/dictBuilder/zdict.h>
 #include <lz4hc.h>
+#include <zdict.h>
```

Comment:
I needed to do the same recently as I have updated LZ4. I am wondering why you are doing the same without an update on the LZ4 commit?!?

Reply:
This file is actually of the `zstd` library rather than the `lz4` library.

Before this PR, we manually included `${PROJECT_SOURCE_DIR}/third_party/zstd`, i.e., only the library's root directory. Now, we include `${ZSTD_LIBRARY_DIR}/dictBuilder` (see `third_party/CMakeLists.txt` line 163) as `PUBLIC`.
Even though this line 163 was already present in the code before, the included directories were `PRIVATE`, i.e., they were only included for the source files of the `zstd` library; they were not populated to `hyrise_impl`. Thus, to include the file `zdict.h`, the manually included directory `${PROJECT_SOURCE_DIR}/third_party/zstd` was used. Now, changing the included directories of the `zstd` library to `PUBLIC`, they are also populated to the source files of `hyrise_impl`.